satpy/readers/tropomi_l2.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2019 Satpy developers
+#
+# This file is part of Satpy.
+#
+# Satpy is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later
+# version.
+#
+# Satpy is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+# A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# Satpy.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Interface to TROPOMI L2 format
+
+"""
```

Comment:
If you have a chance and have the information it would be nice to have more information about this format, what kinds of products it has, what version of the format it was written for, or any other relevant information or links in this description. Maybe the first line of this docstring could be changed to "TROPOMI L2 Reader". We haven't finalized how we want to document every reader in Satpy but this would be a start.